catalog/dags/providers/provider_api_scripts/rawpixel.py

```diff
@@ -250,18 +244,10 @@ def _get_creator(data: dict) -> str | None:
         return source or None
 
     @staticmethod
-    def _get_tags(metadata: dict) -> list[str]:
-        keywords = metadata.get("popular_keywords")
-        if keywords:
-            return [
-                keyword
-                for keyword in keywords
-                if not any(
-                    exclude_tag in keyword
-                    for exclude_tag in RawpixelDataIngester.tags_exclude_list
-                )
-            ]
-        return []
+    def _get_tags(metadata: dict) -> set | None:
+        if keywords := metadata.get("popular_keywords"):
+            return {k for k in keywords}
```

Comment:
```suggestion
            return set(keywords)
```
Would this be simpler?

Reply:
Indeed!